grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -422,18 +420,7 @@ private Json serialiseStateSubset(TaskState state) {
         return Json.object()
                 .set(ID, state.getId().getValue())
                 .set(STATUS, state.status().name())
-                .set(CREATOR, state.creator())
```

Comment:
I deleted a bunch of stuff that wasn't used. If we want recurring and stoppable tasks, we change the redisq library